learn/tanssi/appchain-services/block-production.md

```diff
@@ -0,0 +1,176 @@
+
+When a new session starts, the Tanssi protocol will put the queued assignment into effect. Sequencers will automatically change and start producing blocks in the new Tanssi appchain they've been assigned to while discarding the chain state from the previous assignment. Tanssi will also calculate the new assignment, considering changes in Tanssi appchains that might have been activated or deactivated and block producers that might have been added or removed from the pool or changed the total staked value. This new assignment will be queued for the next session.
+
+![Sessions](/images/learn/tanssi/technical/technical-2.webp)
+
+### The Role of the Tanssi Network {: #tanssi-newtwork }
+
+As previously discussed, the Tanssi protocol assigns sequencers to the Tanssi appchains, and the result of this assignment is stored within the chain state.  Besides running the appchain node, the sequencers also run the Tanssi one. Hence, by accessing the data stored in the finalized blocks of the Tanssi network, they can learn their assignation for the session, and the Tanssi appchains can confirm that a certain group of sequencers have been assigned to them. 
+
+As the Tanssi appchains produce blocks, those blocks need to be validated and finalized by an external security provider. Once an operator verifies a block, a small proof of validity is produced and stored in Tanssi, keeping track of the the proofs for each block of each chain. This small representation of the proof of validity is called [candidate receipt](https://polkadot.network/blog/the-path-of-a-parachain-block#candidate-receipts){target=\_blank} and is composed of a set of values, including the state root, which can be used to verify state proofs.
+
+Finally, Tanssi can verify that the author of an appchain block was the expected one and reward accordingly.
+
+The following diagram shows a simplified model of the data Tanssi stores in its internal state. For every active appchain (in this example, two), Tanssi stores the assigned sequencers, which are the only ones authorized to produce blocks on the appchain's behalf, proof of validity (candidate receipts) extended by the security provider's operators, the latest state root, and the latest block producer. 
+
+```mermaid
+classDiagram
+
+    class tanssi["Tanssi State"] {
+        Appchains Current Sequencer Assignment
+        Appchains Next Session's Sequencer Assignment
+    }
+
+    class appchain1["Appchain 1"] {
+        Current Sequencers assigned
+        Last Block's Proof of Validity
+        Latest State Root
+        Latest Block Producer
+    }
+
+    class appchain2["Appchain 2"] {
+        Current Sequencers assigned
+        Last Block's Proof of Validity
+        Latest State Root
+        Latest Block Producer
+    }
+
+    tanssi-->appchain1
+    tanssi-->appchain2
```

Comment:
I'm not super fond of this Diagram tbh - Maybe we can check with @dawnkelly09 or someone else on how to improve it?